EDL_Code.ino

```diff
@@ -439,6 +441,14 @@ void straight(byte V_REF_L_VALUE, byte V_REF_R_VALUE){
 */
 void stop(){
 	//stop
+	// @Kevin 
+	/*
+	This function stops both motors even if both encoders do not reach desired values.
+	Do we want seperate functions to stop each motor seperately so they both reach the desired encoder values,
+	or just modify this function with inputs of Left, Right, Both. Also we could slow one motors voltage to slow how
+	quickly it reaches its encoder value so that they reach them at similar times (a few options are available)
```

Comment:
I agree that this could be improved. though this is only called after the encoder of the wheel is reached. if we were to use this to directly watch the vrefs and encoder counts you could do that. i think that it might be over kill. give it ago!